### PR TITLE
[zh] Resync pages under working-with-objects

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Review by howieyuen — APPROVED** [review]
/approve

File: content/zh/docs/concepts/overview/working-with-objects/finalizers.md [hunk 1]
```diff
@@ -32,32 +32,37 @@ Kubernetes 自动指定了一些 Finalizers，但你也可以指定你自己的
...
 
 当你使用清单文件创建资源时，你可以在 `metadata.finalizers` 字段指定 Finalizers。
-当你试图删除该资源时，管理该资源的控制器会注意到 `finalizers` 字段中的值，
+当你试图删除该资源时，处理删除请求的 API 服务器会注意到 `finalizers` 字段中的值，
 并进行以下操作：
 
   * 修改对象，将你开始执行删除的时间添加到 `metadata.deletionTimestamp` 字段。
-  * 将该对象标记为只读，直到其 `metadata.finalizers` 字段为空。
+  * 禁止对象被删除，直到其 `metadata.finalizers` 字段为空。
+  * 返回 `202` 作为状态码（HTTP "Accepted"）。
```
Comment (ydFu) [inline content/zh/docs/concepts/overview/working-with-objects/finalizers.md:51]:
```suggestion
  * 返回 `202` 的状态码（HTTP "Accepted"）。
```

Comment (mengjiao-liu) [reply content/zh/docs/concepts/overview/working-with-objects/finalizers.md:51]: ''返回 `202` 的状态码" is indeed more in line with the English source text, but this PR has been merged, I didn't notice this just now, I will mention another PR to fix this.

Comment (ydFu) [reply content/zh/docs/concepts/overview/working-with-objects/finalizers.md:51]: I think it's better.

**Pull request merged**